src/protocols/secure_channel/CASESession.h

```diff
@@ -168,7 +168,44 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
      **/
     void Clear();
 
+    void InvalidateIfPendingEstablishment();
+
 private:
+    class CASESessionFabricDelegate final : public chip::FabricTable::Delegate
```

Comment:
Any reason why we didn't just have `CASESession` just implement the `FabricTable::Delegate` itself?

Reply:
This requires storing an extra pointer width member in the overall ram footprint of `CASESession`. Given that we are in RAM saving mode all over the place in the SDK, we should try to save those 4 (8 bytes on 64-bit platforms) if we can (the effect of which gets multiplied by ~ 15 = 120 bytes of RAM on most platforms...)